client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -295,6 +296,7 @@ export default connect(
 		const siteSlug = getSelectedSiteSlug( state );
 		const isPrivateAtomic =
 			isPrivateSite( state, siteId ) && isSiteAutomatedTransfer( state, siteId );
+		const isJetpack = isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } );
```

Comment:
I'd rename this to `isJetpackNotAtomic` or similar for better clarity. It's very easy to confuse with the general "is Jetpack" scenario that is also true for atomic sites.